docs/database/mongodb.mdx

```diff
@@ -55,9 +55,14 @@ You can access Mongoose models as follows:
 
 ## Using other MongoDB implementations
 
-Limitations with [DocumentDB](https://aws.amazon.com/documentdb/) and [Azure Cosmos DB](https://azure.microsoft.com/en-us/products/cosmos-db):
+Limitations with [DocumentDB](https://aws.amazon.com/documentdb/), [Azure Cosmos DB](https://azure.microsoft.com/en-us/products/cosmos-db) and [Firestore](https://cloud.google.com/products/firestore/mongodb-compatibility):
 
 - For Azure Cosmos DB you must pass `transactionOptions: false` to the adapter options. Azure Cosmos DB does not support transactions that update two and more documents in different collections, which is a common case when using Payload (via hooks).
 - For Azure Cosmos DB the root config property `indexSortableFields` must be set to `true`.
 - The [Join Field](../fields/join) is not supported in DocumentDB and Azure Cosmos DB, as we internally use MongoDB aggregations to query data for that field, which are limited there. This can be changed in the future.
 - For DocumentDB pass `disableIndexHints: true` to disable hinting to the DB to use `id` as index which can cause problems with DocumentDB.
+- For Firestore the recommended options are:
+  - `disableIndexHints: true` to disable hinting to the DB to use `id` as index which can cause problems with Firestore.
+  - `transactionOptions: false` to disable transactions
+  - `compatabilityMode: 'firestore'` to enable Firestore compatibility mode. This does not guarantee full compatability, but solves some common issues such as producing valid sort aggregations and support for custom IDs of type number.
+  - `manualJoins: true` to disable join aggregations and instead populate join fields via multiple queries.
```

Comment:
I don't think this is the best name for this feature flag. I would probably call it `useJoinAggregations` or something. Our flags are usually `use` or `disable` though we haven't been perfectly consistent with this.